src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/HdcService.kt

```diff
@@ -86,9 +85,13 @@ class HdcService(
       error("HDC licence for ${nomisRecord.prisonerNumber} could not be created as they are not approved for HDC")
     }
 
-    if (curfewAddress == null) {
+    if (hdcLicenceData?.curfewAddress == null) {
       error("HDC licence for ${nomisRecord.prisonerNumber} could not be created as there is no curfew address")
     }
+
+    if (hdcLicenceData.curfewTimes == null) {
```

Comment:
I guess this will change when we capture curfew times for HDC licences within CVL